platform/platform-logging/src/main/java/org/codice/ddf/logging/platform/LoggingServiceBeanMBean.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.logging.platform;
+
+import java.util.List;
+
+/**
+ * The {@link LoggingServiceBeanMBean} interface is an MBean 
+ * interface that defines logging event operations.
+ *
+ */
+public interface LoggingServiceBeanMBean {
```

Comment:
`BeanMBean` doesn't look very good. Can't we just name it `LoggingServiceMBean`?